.github/styles/Vocab/Products/accept.txt

```diff
@@ -1,44 +1,81 @@
+721s
 ABIs
 addValidator
 AddValidator
+Agrawal
+aggresive
+Aggresive
+Akanimoh
 Algolia
 alloc
 Alloc
+alloweds
 ANR's
+Ansible
 API's
 APIs
 architecting
+Arweave
+auctionId
 auditable
 autogenerated
 Ava
 Ava Labs
 Avalanche-CLI
+avalanchego
 AvalancheGo
 Avascan
+avax
+Avax
 AVAX
+Berkay
+Binance
+bintools
+Blanco
+blackhole
```

Comment:
I think this had been fixed by another PR merged recently.

Reply:
I don't see that. Which PR do you mean?

Reply:
I see it here. https://github.com/ava-labs/avalanche-docs/pull/1115

Removing from `accept.txt` and resolving this comment.